### PR TITLE
feat:implemented power function

### PR DESCRIPTION
**Pull request by albin1616**
implemented power function

### CODE REVIEW

File: power.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function power(a,b){
```
Comment (Rajharsh2) [inline power.js:1]: please add jsdoc

**Review by Rajharsh2 — CHANGES_REQUESTED** [review]

File: power.js (hunk 1 above)
Comment (albin1616) [reply power.js:7]: resolved

File: power.js [hunk 2]
```diff
@@ -1,3 +1,9 @@
+/**
+ * 
+ * @param {number} a must be a base
+ * @param {*} b must be exponent
+ * @returns a result 
+ */
 function power(a,b){
     let product=a;
```
Comment (Rajharsh2) [inline power.js:8]: put comments also for better exlpanation.

**Review by Rajharsh2 — CHANGES_REQUESTED** [review]

File: power.js (hunk 2 above)
Comment (albin1616) [reply power.js:8]: done

**Review by Rajharsh2 — APPROVED** [review]

File: power.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /**
- * 
+ * this function is used implementing power function
  * @param {number} a must be a base
  * @param {*} b must be exponent
```
Comment (manvithalva12) [inline power.js:4]: give explaination

**Review by manvithalva12 — CHANGES_REQUESTED** [review]

File: power.js (hunk 3 above)
Comment (albin1616) [reply power.js:4]: done

**Review by manvithalva12 — APPROVED** [review]

**Comment by manvithalva12** [comment]
lgtm

File: power.js [hunk 4]
```diff
@@ -10,4 +10,5 @@ function power(a,b){
         product=product*product;
     }
     return;
-}
\ No newline at end of file
+}
```
Comment (Rajharsh2) [inline power.js:13]: add comment to above the fnction

**Review by Rajharsh2 — CHANGES_REQUESTED** [review]

**Review by Rajharsh2 — APPROVED** [review]

**Pull request merged**